graphql_api/types/bundle_analysis/base.py

```diff
@@ -135,6 +145,65 @@ def resolve_assets(
     return list(bundle_report.assets())
 
 
+@bundle_report_bindable.field("assetsPaginated")
+def resolve_assets_paginated(
+    bundle_report: BundleReport,
+    info: GraphQLResolveInfo,
+    ordering: AssetOrdering = AssetOrdering.SIZE,
```

Comment:
should this and ordering_direction be optional as well since we're defaulting a value?

Reply:
yes, good catch

Reply:
wait I think it is currently ok, right now it should mean that `ordering` is of type `AssetOrdering`, but if that param is not passed in to the call it would be defaulted to `AssetOrdering.SIZE`. It should never be the case that `ordering` is null.

Reply:
Fair enough, I can appreciate the defensive nature of having a fallback value here